shell/platform/android/io/flutter/plugin/common/MessageCodec.java

```diff
@@ -26,6 +26,12 @@
   /**
    * Decodes the specified message from binary.
    *
+   * <p><b>Warning:</b> The ByteBuffer may be `direct` if `wantsDirectByteBufferForDecoding` returns
```

Comment:
Is this right that this is the only place in this PR that mentions `wantsDirectByteBufferForDecoding`?

Reply:
Done.